training_pipeline/models/README.md

```diff
@@ -2,7 +2,8 @@
 
 - `train.py`: the `run_fn` function inside this file is designated in the `run_fn` argument of the `Trainer` component. `run_fn` function basically constructs `tf.data` pipeline for training and validation dataset, builds a model, train the model, and export the trained model. 
   - `unet.py`: the model used in this project is a **U-NET** variant borrowed from [TensorFlow's official tutorial](https://www.tensorflow.org/tutorials/images/segmentation). the `build_model` function returns the U-NET implementation based on Keras, and it is used in the `train.py`.
-  - `signatures.py`: a signature let us add custom functions to a model when it is exported as `SavedModel`. This file defines a set of signatures for the model.
+  - `signatures.py`: a signature let us add custom functions to a model when it is exported as `SavedModel`. This file defines a set of signatures for the model. `model_exporter` defines a signature that handles requests when the model is deployed, `transform_features_signature` defines a signature that applies preprocessing logics from the `Transform` component to the input, and `tf_examples_serving_signature` defines a signature that gets predictions from the model after applying the preprocessing logics from the `Transform` component. 
+    - NOTE: difference between `model_exporter` and `tf_examples_serving_signature` is whether the preprocessing logics from the `Transform` component is applied.
```

Comment:
Is there a flag within the utilities that controls this behaviour? If so, let's list them down.

Reply:
sure!